superset/views/database/forms.py

```diff
@@ -133,129 +124,136 @@ class CsvToDatabaseForm(UploadToDatabaseForm):
-        ),
-        validators=[Optional(), NumberRange(min=0)],
-        widget=BS3TextFieldWidget(),
-    )
-    mangle_dupe_cols = BooleanField(
-        _("Mangle Duplicate Columns"),
-        description=_('Specify duplicate columns as "X.0, X.1".'),
-    )
-    usecols = JsonListField(
-        _("Use Columns"),
-        default=None,
-        description=_(
-            "Json list of the column names that should be read. "
-            "If not None, only these columns will be read from the file."
-        ),
-        validators=[Optional()],
-    )
-    skipinitialspace = BooleanField(
-        _("Skip Initial Space"), description=_("Skip spaces after delimiter.")
-    )
-    skiprows = IntegerField(
-        _("Skip Rows"),
-        description=_("Number of rows to skip at start of file."),
-        validators=[Optional(), NumberRange(min=0)],
-        widget=BS3TextFieldWidget(),
-    )
-    nrows = IntegerField(
-        _("Rows to Read"),
-        description=_("Number of rows of file to read."),
-        validators=[Optional(), NumberRange(min=0)],
-        widget=BS3TextFieldWidget(),
+    skip_initial_space = BooleanField(
+        _("Skip Initial Space"), description=_("Skip spaces after delimiter")
     )
     skip_blank_lines = BooleanField(
         _("Skip Blank Lines"),
-        description=_("Skip blank lines rather than interpreting them as NaN values."),
+        description=_(
+            "Skip blank lines rather than interpreting them as Not A Number values"
+        ),
     )
     parse_dates = CommaSeparatedListField(
-        _("Parse Dates"),
+        _("Columns To Be Parsed as Dates"),
         description=_(
-            "A comma separated list of columns that should be parsed as dates."
+            "A comma separated list of columns that should be parsed as dates"
         ),
         filters=[filter_not_empty_values],
     )
     infer_datetime_format = BooleanField(
-        _("Infer Datetime Format"),
-        description=_("Use Pandas to interpret the datetime format automatically."),
+        _("Interpret Datetime Format Automatically"),
+        description=_("Interpret the datetime format automatically"),
     )
     decimal = StringField(
         _("Decimal Character"),
         default=".",
-        description=_("Character to interpret as decimal point."),
+        description=_("Character to interpret as decimal point"),
         validators=[Optional(), Length(min=1, max=1)],
         widget=BS3TextFieldWidget(),
     )
-    index = BooleanField(
-        _("Dataframe Index"), description=_("Write dataframe index as a column.")
+    null_values = JsonListField(
+        _("Null Values"),
+        default=config["CSV_DEFAULT_NA_NAMES"],
+        description=_(
+            "Json list of the values that should be treated as null. "
+            'Examples: [""], ["None", "N/A"], ["nan", "null"]. '
+            "Warning: Hive database supports only single value"
```

Comment:
nit typo in existing text.. should this be "only a single value"?

Reply:
Oh Thanks! Fixed